### ISSUE TITLE
Setup browser-based scraping as fallback

### ISSUE THREAD

**Issue by joemarct** [Volentix/venue-server#210]
Crawlera only provides proxies and that will only serve us its purpose if scraping fails because bitcointalk blocked our IP. If bitcointalk uses cloudflare to detect and block scraper bots (which I have personally seen a few times), Crawlera proxies cannot help us.

Crawlera provides new proxy IPs on demand, it's still up to us to use these proxies to send direct HTTP requests or use them behind a browser.

In the spirit of designing a more resilient scraping system, I propose that we use these proxies behind a browser using selenium. That way, our fallback circumvents both IP blocking and Cloudflare's scraper bot blocking.

Here's a guide on how to use Cloudflare with Selenium:
https://support.scrapinghub.com/support/solutions/articles/22000203564-using-crawlera-with-selenium-and-polipo

**Comment by shawnlauzon** [comment]
@alexdashkov You suggested Crawlera, can you provide feedback on this issue?

**Comment by alexdashkov** [comment]
I use crawlera a lot and didn't see the blocks described in this issue, however, I used it using scrapy (with retry policy and user-agent rotation). 

@shawnlauzon can you check crawlera stats? Do you have a lot of bad requests there? 

Of course, we can set up selenium, but I'm not sure that it's something that is required right now.

**Comment by shawnlauzon** [comment]
I cannot check it right now. You should have access to the system, could
you check it?

**Comment by alexdashkov** [comment]
So right now I don't see any request in the crawlera.

**Comment by shawnlauzon** [comment]
As of today, I do see requests

![image](https://user-images.githubusercontent.com/611529/44623168-17a43b80-a896-11e8-93b0-398044dfc6d4.png)

**Comment by alexdashkov** [comment]
It goes from my local tests

**Comment by alexdashkov** [comment]
No, I didn’t mean that this project should use scrapy. I meant that I
didn’t see the problems described in this issue.

**Comment by shawnlauzon** [comment]
Summary of this seems to be that we don't need to implement this unless we see a problem with the existing fallback solution of purely crawlera. Moving to Backlog
